services/BackgroundRunnerService.ts

```diff
@@ -0,0 +1,121 @@
+import { log } from "@ledgerhq/logs";
+import { withDevicePolling } from "@ledgerhq/live-common/lib/hw/deviceAccess";
```

Comment:
I dont understand why we have both ts and js files, shouldn't the js ones be deleted?
Can you ping Luiz on it to confirm? Otherwise, I guess all changes need to happen on ts files only, and then rebuild

Reply:
Done